docs/pages/tutorial/eas/environment-variables.mdx

```diff
@@ -0,0 +1,252 @@
+You can use the Export option on the [website](https://expo.dev/accounts/[account]/projects/[project]/environment-variables) as well to download the file.
+
+After running
+
+<Terminal
+  cmd={['$ eas env:pull --environemnt development']}
+  cmdCopy="eas env:pull --environemnt development"
+/>
+
+command the **.env** file will be created in the root of your project with the following content:
+
+```bash .env.local
+# Environment: development
+
+APP_VARINT=development
+EXPO_PUBLIC_API_URL=https://staging.my-api-url.mycompany.com
+# GOOGLE_SERVICES_JSON=***** (secret variables are not available for reading)
+SENTRY_AUTH_TOKEN=token
+```
+
+The downloaded `EXPO_PUBLIC_` variables are available for local development when using `npx expo start` command. The `GOOGLE_SERVICES_JSON` variable is not available to be pulled to the local environment as it is a secret variable.
+
+> **warning** It is best to add all of the **.env** files to the **.gitignore** to avoid committing it to the repository and exposing the sensitive information. Commiting the **.env** file may additionally lead to environment variables resolution conflicts.
+
+</Step>
+
+<Step label="4">
+## Use EAS environment variables with EAS Build
+
+To have a full control over the environments used for your builds, you can specify the [`environemnt`](/eas/json#environemnt) field in the build profiles settings in the `eas.json` file.
+
+```json eas.json
+{
+  "build": {
+    "production": {
+      /* @info Using <CODE>production</CODE> environemnt */
+      "environment": "production"
+      /* @end */
+      /* @hide ... */ /* @end */
+    },
+    "preview": {
+      /* @info Using <CODE>preview</CODE> environemnt */
+      "environment": "preview"
+      /* @end */
+      /* @hide ... */ /* @end */
+    },
+    "development": {
+      /* @info Using <CODE>development</CODE> environemnt */
+      "environment": "development"
+      /* @end */
+      /* @hide ... */ /* @end */
+    },
+    "my-profile": {
+      /* @info Using <CODE>production</CODE> environemnt */
+      "environment": "production"
+      /* @end */
+      /* @hide ... */ /* @end */
+    }
+  }
+}
+```
+
+All of the environment variables from the selected environemnt will be used during the build process.
+Plain text and sensitive variables will be available when resolving build configuration based on the dynamic app config in EAS CLI as well.
+
+> **info** The secret type environemnt variables are not available during build configuration resolution in EAS CLI as they are not readable outside of the EAS servers.
+
+In case of our example app, all of the environment variables except `GOOGLE_SERVICES_JSON` (which is secret) will be available during build configuration resolution in EAS CLI.
+
+It's especially important in this context to have the correct visibility set for `APP_VARIANT` variable in order for it to be available in EAS CLI so it is able to generate credentials for the correct bundle identifier and package name.
```

Comment:
```suggestion
It's especially important in this context to have the correct visibility set for the `APP_VARIANT` variable so that it's available in EAS CLI so it is able to generate credentials for the correct bundle identifier and package name.
```